### ISSUE TITLE
Integrate FERC Market Based Rate MBR ownership data

### ISSUE THREAD

**Issue by zaneselvans** [catalyst-cooperative/pudl#2314]
When someone purchases a power plant and wants to sell power at market based rates, they are required to disclose their internal ownership structure, even when they are **private equity** firms who don't disclose this information anywhere else! Unfortunately, accessing this data is pretty manual.

Data about the MBR applications can be downloaded kind of in bulk via https://mbrweb.ferc.gov/  This includes the docket numbers in which the PE ownership information can be found. However, to download the actual PDFs from those dockets you have to use the [FERC eLibrary](https://elibrary.ferc.gov/) which is a horrendous GUI.

This ownership information would supplement the Ex-21 data from the SEC-10K (see #2225) and the [EIA-860 ownership data](https://data.catalyst.coop/pudl/ownership_eia860), but actually accessing it would require some kind of automated docket downloading and PDF parsing system.

**Comment by zaneselvans** [comment]
@jrea-rmi @arengel I dunno if this private equity generation asset ownership data might be something y'all are interested in. Not sure how much of the overall fleet it would impact.

**Comment by jrea-rmi** [comment]
Yes this would be interesting! Would it be additional or redundant to ownership in EIA-860?

**Comment by zaneselvans** [comment]
It's definitely information that doesn't exist in EIA-860. `ownership_eia860` includes the name of the company that owns the generator slice, but who owns *that* company is unclear. If it's a publicly traded company, the SEC 10-K / Ex-21 data will say, but if it's a private equity firm, it will be totally opaque.  But **FERC** does have that information it turns out!

**Comment by jrea-rmi** [comment]
Awesome. The publicly traded companies are higher priority for our current projects. But we would use both for internal decisions now, and potentially in the future for new projects too!

**Comment by zaneselvans** [comment]
E.g. there's a company named "Lightstone" which bought the Gavin coal plant from AEP. You can see in the ownership table that Lightstone owns the plant, but who the heck is Lightstone? It turns out it's owned by several private equity funds that ultimately trace back to Blackrock and ArcLight, based on information in Lightstone's filing in docket ER17-652-000.

**Comment by zaneselvans** [comment]
@MichaelTiemannOSC I dunno if this is a kind of ownership data y'all have looked at before.

**Comment by jrea-rmi** [comment]
On the Gavin coal plant, interesting that S&P says it's owned by Arclight and Blackstone

![image](https://user-images.githubusercontent.com/56084640/219711231-a4c9d334-e927-4f10-a850-6edabf469254.png)

**Comment by jrea-rmi** [comment]
This kind of discrepancy is common in my experience, not sure where S&P gets all its information. They say Lightstone Generation is owned by the owners the list on the Gavin Plant profile:

![image](https://user-images.githubusercontent.com/56084640/219717554-35e5710b-f433-40dd-b429-5777b11702a0.png)

but don't have ownership information on Gavin Power, LLC:

![image](https://user-images.githubusercontent.com/56084640/219718004-578eb29a-ef55-4c20-b99a-d0c189ff927f.png)
